web/src/sidebar_ui.ts

```diff
@@ -191,9 +191,7 @@ export function initialize_left_sidebar(): void {
 }
 
 export function initialize_right_sidebar(): void {
-    const rendered_sidebar = render_right_sidebar({
-        realm_rendered_description: page_params.realm_rendered_description,
-    });
+    const rendered_sidebar = render_right_sidebar();
```

Comment:
There's surely some CSS to remove too:

`web/styles/right_sidebar.css:.right-sidebar-shortcuts .realm-description {`

Reply:
Done